module/VuFind/tests/unit-tests/src/VuFindTest/RecordDriver/Feature/MarcAdvancedTraitTest.php

```diff
@@ -204,4 +204,116 @@ public function testMarcAdvancedTraitAltScript(): void
             $obj->getTitleSectionsAltScript()
         );
     }
+
+    /**
+     * Test getMarcFieldWithInd when a single indicator value is sent
+     *
+     * @return void
+     */
+    public function testGetMarcFieldWithIndOneValue(): void
+    {
+        $obj = $this->getMockDriverFromFixture('marc/marctraits.xml');
+
+        // Test when a single ind value is passed
+        $this->assertEquals(
+            ['upc'],
+            $obj->getMarcFieldWithInd('024', null, [['1' => ['1']]])
+        );
+    }
+
+    /**
+     * Test getMarcFieldWithInd when multiple values for the indicator are sent
+     *
+     * @return void
+     */
+    public function testGetMarcFieldWithIndTwoValues(): void
+    {
+        $obj = $this->getMockDriverFromFixture('marc/marctraits.xml');
+
+        // Test when multiple values for the same ind are passed
+        $this->assertEquals(
+            ['upc', 'ismn'],
+            $obj->getMarcFieldWithInd('024', null, [['1' => ['1', '2']]])
+        );
+    }
+
+    /**
+     * Test getMarcFieldWithInd when multiple indicators are requested
+     * as OR conditions
```

Comment:
I think the comments are reversed on this test and the following one -- it says OR here instead of AND, and vice versa.